app/assets/javascripts/pageflow/editor/views/change_theme_view.js

```diff
@@ -0,0 +1,35 @@
+pageflow.ChangeThemeView = Backbone.Marionette.ItemView.extend({
+  template: 'templates/change_theme',
+
+  ui: {
+    changeThemeButton: '.change_theme',
+    labelText: 'label .name'
+  },
+
+  events: {
+    'click .change_theme': function() {
+      pageflow.ChangeThemeDialogView.open({
+        model: this.model,
+        themes: this.options.themes
+      });
+    }
+  },
+
+  onRender: function() {
+    this.ui.labelText.text(this.labelText());
+  },
+
+  labelText: function() {
+    return this.options.label || this.labelPrefix() + ': ' + this.localizedThemeName();
```

Comment:
Of course it would be nice to generate the preview images using actual media files from the entry, but that would tax computer and developer significantly